doc/conf.py

```diff
@@ -12,13 +12,14 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-# Get configuration information from setup.cfg
 from configparser import ConfigParser
-conf = ConfigParser()
+from importlib import metadata, resources
 
-import os
-conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
-setup_cfg = dict(conf.items('metadata'))
+# Get configuration information from setup.cfg
+pypeit_root = resources.files("pypeit").parent
+conf = ConfigParser()
+conf.read([pypeit_root / "setup.cfg"])
```

Comment:
Nice catch on this!

Reply:
Turns out this was wrong.  The `resources.files()` looks in the installed package directory (if running in CI, is somewhere in a `site-packages`) rather than the source code directory.  For editable installations (_e.g._, `pip install -e .`), these amount to the same thing, which is why I didn't catch it initially.